components/ILIAS/Mail/classes/class.ilMail.php

```diff
@@ -506,9 +557,10 @@ private function sendInternalMail(
         $usr_id = $usr_id ?: $this->user_id;
 
         if ($use_placeholders) {
-            $message = $this->replacePlaceholders($message, $usr_id);
+            $message = $this->replacePlaceholders($message, $usrId);
         }
-        $message = str_ireplace(['<br />', '<br>', '<br/>'], "\n", $message);
+        $message = $this->formatLinebreakMessage($this->refinery->string()->markdown()->toHTML()->transform($message) ?? '');
+        $message = str_ireplace(["<br />", "<br>", "<br/>"], "\n", $message);
```

Comment:
We should not change the format:

```suggestion
        $message = str_ireplace(['<br />', '<br>', '<br/>'], "\n", $message);
```

But: Do we really still need to replace the HTML linebreaks, if we transform `$message` with `$this->refinery->string()->markdown()->toHTML()->transform(...)`?

Reply:
is now removed